packages/react-components/react-swatch-picker-preview/stories/SwatchPicker/SwatchPickerBestPractices.md

```diff
@@ -0,0 +1,24 @@
+## Best practices
```

Comment:
Let's wrap this with `<details>` & `<summary>` like for other components.

Reply:
fixed